client/state/billing-transactions/actions.js

```diff
@@ -44,6 +47,31 @@ export const requestBillingTransactions = () => {
 	};
 };
 
+export const requestBillingTransaction = receiptId => dispatch => {
+	dispatch( {
+		type: BILLING_RECEIPT_REQUEST,
+		receiptId,
+	} );
+
+	return wp
+		.undocumented()
+		.me()
+		.getReceipt( receiptId, { format: 'display' } )
+		.then( receipt => {
+			dispatch( {
+				type: BILLING_RECEIPT_REQUEST_SUCCESS,
+				receiptId,
+				receipt,
+			} );
+		} )
+		.catch( () => {
+			dispatch( {
+				type: BILLING_RECEIPT_REQUEST_FAILURE,
```

Comment:
With this, we're treating request errors (like timeouts) the same as permission errors (trying to view someone else's receipt). What do you think about differentiating between the two?

If the receipt request fails due to timeout - should we show a notice with a retry action?